_includes/faqs-query-updatedate.html

```diff
@@ -0,0 +1,16 @@
+
+  <div class="api-explorer">
+    <div class="query">
+    </div>
+    <div class="explorer">
+<div class="api-explorer-form">
+  <textarea class="form-control api-explorer-query autosize" rows="1" style="display:none;">https://api.trade.gov/v2/ita_faqs/search?api_key={{ site.apikey }}&update_date=2014-01-01 TO 2014-11-01</textarea>
+  <button class="btn go pull-left" type="button">http://api.trade.gov/v2/ita_faqs/search?update_date=2014-01-01 TO 2014-11-01</button>
+</div>
+<div class="return">
+  <div class="response api-explorer-section">
+    <pre><span class="default"></span></pre>
+  </div>
+</div>
+    </div>
+  </div>
```

Comment:
All of these `api-explorer` chunks are so similar. The part that's different from one to the next is the endpoint URL and query string. Is there any way to move this HTML into a shared template, so that it's not repeated so many times?